skyportal/handlers/api/instrument.py

```diff
@@ -48,6 +49,9 @@ def post(self):
         )
 
         sensitivity_data = data.get("sensitivity_data", None)
+        if isinstance(sensitivity_data, str):
+            sensitivity_data = ast.literal_eval(sensitivity_data.replace("\'", "\""))
```

Comment:
If you really only want a dictionary, maybe `json.loads` is more restrictive (in a good way)?